docs/superpowers/plans/2026-08-19-sp4c2b-no-victim-player-turn.md

```
@@ -0,0 +1,684 @@
+
+```ts
+/** SP-4c-2b: no victim this turn — an ally-targeted cast resolved nobody on the opposing side.
+ *  Every victim-derived read below must answer "there is no enemy", NEVER "an enemy with neutral
+ *  stats": the latter is exactly the dummy ghost this rung deletes (see plan §A.4-A.5 — the ghost's
+ *  `shieldPool` was arming `enemyShielded` gates in 22 support-ship fingerprints). */
+const hasVictim = enemy !== undefined;
+```
+
+| class | sites in `playerTurn.ts` | the no-victim answer |
+| --- | --- | --- |
+| **1. Gate context** | `1612-1614`, `1643`, `2042`, `2152`, `2420-2422`, `2454` | **Omit the field**, via TWO helpers — see the ⚠️ below. | All are already optional with documented defaults: `targetSpeed?`/`targetCurrentHp?`/`targetCritPower?` (`roundContext.ts:101,160` → `?? 0`; `evaluateConditions.ts:92,199`), `enemyShielded?` (`triggers.ts:1779` → default `false`). Use the conditional-spread idiom `buildTurnArgs` already uses for `targetId`. |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)